[ML-Commons]

lchangxii opened the pull request:
This pull request is from MICRO2023 Artifact, ID 85.

LChangxi committed:
[ML-Commons]

gfursin reviewed (approved):
Checking now ... Thanks!

gfursin commented:
Thank you. Your CM interface is fine but I simplified it and moved your scripts and Docker file to script::reproduce-paper-m2023-85 . Please, check it out if you have time! But it's already enough for the pilot project!

lchangxii commented:
> Thank you. Your CM interface is fine but I simplified it and moved your scripts and Docker file to script::reproduce-paper-m2023-85 . Please, check it out if you have time! But it's already enough for the pilot project!

I checked it. Everything is fine. Thanks for your help.
